src/account-info/LogoutButton.js

```diff
@@ -39,9 +38,9 @@ class LogoutButton extends PureComponent<Props> {
   };
 
   logout = () => {
-    const { accounts, actions } = this.props;
+    const { actions, auth } = this.props;
     this.shutdownPUSH();
-    actions.logout(accounts);
```

Comment:
Good to fix this -- this is a great example of where types would have helped us out.

Looking at `yarn run flow coverage --color src/account-info/LogoutButton.js` before this change, we don't have coverage on `actions.logout` -- which is because the `Actions` type is untyped.

Reply:
Also I would go further than what you say in the commit message:

> No need to pass accounts as they are not used.

It's not just "no need" -- this is a type error. It's one that won't directly cause a failure at runtime, but it's a clear mismatch between the types someone was thinking writing this call and the types someone was thinking writing the function itself. And that mismatch could easily have caused a bug in itself, and moreover it makes it confusing for anyone reading the code, which causes more bugs.

So, again, good to have fixed. :-)